### ISSUE TITLE
Scanning commercial cells

### ISSUE THREAD

**Issue by Carcalmoa** [free5G/free5GRAN#24]
I've been trying to scan commercial cells with free5GRAN, but I've encountered some problems. 

Since the only information I have about the cells is the frequency range in which they transmit, I have tried to detect them using BAND_SCAN. The problem is that I don't know the ssb_period of the cells. I have tried to use a script in which I iterate different values from 5ms to 160ms with a step of 5ms. It runs for hours, but sometimes I'm lucky and the program is able to detect them. However, the ssb_period in which the scan succeeds seems to be random as each execution of the script detects the cell with different ssb_period values. Is there a way to know the ssb_period in advance? 

I have also tried the SEARCH_CELL and it works to find my own gNBs, but for commercial cells as I don't have any information about ssb_frequency, gscn or ssb_period I find it impossible. 

Is there a more effective way to detect commercial cells? Or is there a way to obtain the values I need to refine my search?

Thank you very much!

**Comment by johst-source** [comment]
Correct me if I'm wrong, but can we not assume that the SSB period is 20ms during inital cell search?

**Comment by Carcalmoa** [comment]
I'm not sure about that, have you been able to detect different commercial cells without having any context information in advance, always using 20ms as the SSB period?

**Comment by johst-source** [comment]
No, am currently unable to locate any commercial cell on any band.

**Comment by desimetallica** [comment]
I'm able to decode MIB and DCI using this tool from a commercial cell. Unfortunately I found ## Cell barred: 1 and SIB decoding does not produce anything in my log.
The cell_scan function was set to  ssb_period: 0.02;
Honestly I don't know why cell is barred because looks like the smartphones are still able to connect with at it.
